### PR TITLE
Support task type in e2e

### PR DESCRIPTION
**Pull request by axinging**
Task type:
performance, run the performance task.
correctness, run the correctness task.

Example usage:
http://localhost:8080/local-benchmark/?backend=webgl&warmup=0&run=2&benchmark=bodypix&inputsize=0.25&architecture=MobileNetV1&inputType=image&task=correctness

FEATURE

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4722)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao PTAL.

File: e2e/benchmarks/local-benchmark/index.html [hunk 1]
```diff
@@ -142,8 +148,12 @@ <h2>TensorFlow.js Model Benchmark</h2>
       if (backendsController != null && urlState.has('backend'))
         backendsController.setValue(urlState.get('backend'));
       // Only auto run when URI component exists.
-      if (runButton != null)
+      if (correctnessButton != null && task == 'correctness') {
+        correctnessButton.domElement.click();
+      }
+      if (runButton != null && task == 'performance') {
```
Comment (qjia7) [inline e2e/benchmarks/local-benchmark/index.html:154]: nit: use `else if`?

**Review by qjia7 — COMMENTED** [review]
LGTM. Would like @lina128 to take a look. Thanks.

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4722#-:-MUExXf07RxliNaBGfZ_:bb6ywmx)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @qjia7)

___
*[e2e/benchmarks/local-benchmark/index.html, line 154 at r1](https://reviewable.io/reviews/tensorflow/tfjs/4722#-MU7UbaLtDh1ThvObGYm-r1-154:-MUExTj_2GSxm58O3e0D:b32v57) ([raw file](https://github.com/tensorflow/tfjs/blob/d968b0822e9cdf0cc97b7139772a8747e4d5ab0e/e2e/benchmarks/local-benchmark/index.html#L154)):*
<details><summary><i>Previously, qjia7 (Jiajia Qin) wrote…</i></summary><blockquote>

nit: use `else if`?
</blockquote></details>

+1

**Pull request merged**